data/apps.csv

```diff
@@ -1430,7 +1430,6 @@ terminal,extraterm,https://extraterm.org/,https://github.com/sedwards2009/extrat
 terminal,st,https://st.suckless.org/,https://git.suckless.org/st/,A simple terminal implementation for X.
 terminal,wterm,,https://github.com/TornadoCookie/wterm,A native Wayland terminal emulator based on an st fork using wld.
 terminal,mlterm,https://mlterm.sourceforge.net/,https://github.com/arakiken/mlterm,"A very fast low latency terminal emulator with features such as rendering variable width fonts, proper bidirectional support out of the box, a daemon mode, multiple XIM, and true background transparency."
-games,sshattrick,,https://github.com/ricott1/sshattrick,Play Hattrick in your terminal over SSH.
```

Comment:
I'm unsure why this one was removed. A conflict issue? Or maybe I'm missing something

Reply:
Yeah, it was by accident